code/convertTDcodes.m

```diff
@@ -0,0 +1,116 @@
+function TDsettings = convertTDcodes(TDdata)
+%%
+% Conversion of Medtronic numeric codes into values (e.g. Hz)
+%
+% Input: Data within "DeviceSettings.SensingConfig.timeDomainChannels",
+% contained in DeviceSettings.json (after converted to table format)
+%
+% Output: Table of the same size as input, with coded values replaced
+%
+% This function is called from createDeviceSettingsTable.m
+% Assumes no bridging across channels
+%%
+
+TDsettings = TDdata;
+for iChan = 1:length(TDsettings)
+    % LFP 1 (front end)
+    switch TDdata(iChan).lpf1
+        case 9
+            TDsettings(iChan).lpf1 = '450Hz';
+        case 18
+            TDsettings(iChan).lpf1 = '100Hz';
+        case 36
+            TDsettings(iChan).lpf1 = '50Hz';
+        otherwise
+            TDsettings(iChan).lpf1 = 'unexpected';
```

Comment:
Are there only 3 options for the first low pass filter?

Reply:
yes